internal/runtime/init/init.go

```diff
@@ -162,6 +172,19 @@ func LoadPaths(paths []string,
 	return &result, nil
 }
 
+// Splits the paths in 2 groups. Ones with .tar.gz and another with
+// non .tar.gz extensions.
+func splitByTarGzExt(paths []string) (targzs []string, nonTargzs []string) {
```

Comment:
Nit: Can we update the comment to something like `splitByTarGzExt splits the paths in 2 groups.....`

Reply:
Ok. will update the comment.  BTW, What does "Nit" mean ? was just curious. 😄

Reply:
Nit refers to "nitpick" which means to point out small trivial issues 🙂

Reply:
@ashutosh-narkar  I have changed the function doc and squashed my commits.